app/services/bulk_tagging/fetch_tagged_content.rb

```diff
@@ -12,6 +12,30 @@ def initialize(content_id, document_type)
     end
 
     def call
+      if taxon_document_type?
+        request_tagged_content_to_taxon
+      else
+        request_tagged_content_items
+      end
+    end
+
+  private
+
+    def taxon_document_type?
+      document_type == "taxon"
+    end
+
+    def request_tagged_content_to_taxon
```

Comment:
Cool, using `get_linked_items` seems better than using `get_expanded_links`. Could you change the current implementation to use it too?